shotover-proxy/examples/windsock/cassandra.rs

```diff
@@ -90,10 +95,6 @@ impl Bench for CassandraBench {
                     Shotover::ForcedMessageParsed => "forced-message-parsed".to_owned(),
                 },
             ),
-            // TODO: run with different OPS values
-            ("OPS".to_owned(), "unlimited".to_owned()),
-            // TODO: run with different connection count
-            ("connections".to_owned(), "128".to_owned()),
```

Comment:
What was the plan for connections?

Reply:
how many concurrent connections the operations are distributed over.
Might end up bringing it back eventually, not quite sure yet how to best decide the number of connections.